src/plugins/intel_cpu/src/nodes/kernels/scaled_attn/softmax_kernel.hpp

```diff
@@ -585,7 +586,16 @@ inline void scale_add2_reduce_max(ov::float16* a,
         }
 
         if (has_attn_mask) {
-            svfloat16_t v_mask = svld1_f16(pg_f16, reinterpret_cast<const float16_t*>(attn_mask + i));
+            svfloat16_t v_mask;
+            if constexpr (std::is_same_v<T, float>) {
+                svbool_t pg_f32 = svwhilelt_b32(i, i + vec_len_f16_neon);
```

Comment:
This is SVE code. Shouldn't we use SVE specific constants like `vec_len_f16_sve` and `vec_len_f32_sve` ?
Also, in the case of f32 attention_mask, it looks like we have to convert f32 in two steps as the fp16 SIMD vector capacity 2x of fp32 vector capacity and narrowing conversion happens leaving the upper word in each double word undefined (just like it's done for neon).